gensim/examples/dmlcz/sources.py

```diff
@@ -190,13 +187,13 @@ class DmlCzSource(DmlSource):
 
     See the ArticleSource class for general info on sources.
     """
+
     def idFromDir(self, path):
         assert len(path) > len(self.baseDir)
         dmlczId = open(os.path.join(path, 'dspace_id')).read().strip()
-        pathId = path[len(self.baseDir) + 1 : ]
+        pathId = path[len(self.baseDir) + 1:]
```

Comment:
This is against PEP8 (see https://www.python.org/dev/peps/pep-0008/#other-recommendations ).